tests/lit-tests/float16_uniform.ispc

```diff
@@ -0,0 +1,78 @@
+//; RUN: %{ispc} %s --target=host --emit-llvm-text -O0 -o - | FileCheck %s
+//; REQUIRES: X86_ENABLED
```

Comment:
Why does this require x86?

Reply:
This should have been removed when I added gen support - removed now